sql/src/main/java/io/crate/planner/node/dql/MergePhase.java

```diff
@@ -60,16 +62,31 @@
      */
     @Nullable private PositionalOrderBy positionalOrderBy;
 
+    /**
+     * Creates a MergePhase to combine the results from multiple node operations.
+     * @param jobId The JobID of the entire execution.
+     * @param executionNodeId A unique execution id for this phase.
+     * @param name The name of the MergePhase.
+     * @param numUpstreams The number of nodes to expect data from.
+     * @param numInputs The number of different inputs to read data from.
+     * @param executionNodes The nodes where this MergePhase executes.
+     * @param inputTypes The types of the input rows.
+     * @param projections The projections to apply when merging.
+     * @param distributionInfo The default strategy to use when distributing the results of the MergePhase.
+     * @param positionalOrderBy The order by positions to apply if ordering is supplied.
```

Comment:
Maybe rephrase this to something like `The order by positions the input is pre-sorted on; setting this will result in a sorted merge`